src/components/sidenav/sidenav-theme.scss

```diff
@@ -1,3 +1,5 @@
 md-sidenav.md-THEME_NAME-theme {
-  background-color: '{{background-color}}';
+  &, md-content {
+    background-color: '{{background-hue-1}}';    
```

Comment:
it shouldn't, take a look at [themes at Material spec](https://www.google.com/design/spec/style/color.html#color-themes)

![image](https://cloud.githubusercontent.com/assets/6004537/13464459/02c75722-e09a-11e5-8fed-b530fdb35283.png)

but i do agree that hue-1 is not the wanted color